argilla/src/argilla/records/_io/_datasets.py

```diff
@@ -39,7 +39,7 @@ def _is_hf_dataset(dataset: Any) -> bool:
         return isinstance(dataset, HFDataset)
 
     @staticmethod
-    def to_datasets(records: List["Record"]) -> HFDataset:
+    def to_datasets(records: List["Record"], schema: Dict) -> HFDataset:
```

Comment:
Maybe more flexible and easier to extend if we pass the whole rg.Dataset, even if internally we only use the schema.

```suggestion
    def to_datasets(records: List["Record"], dataset: "Datasert") -> HFDataset:
```